CHANGELOG.md

```diff
@@ -6,6 +6,10 @@
 
 ### Fixed
 - MSSQL - Prevent potential bad database connection using upgrading an old version of `tedious` dependency.
+- Onboarding - Catch correctly authentication error on mongodb.
+- Onboarding - Catch correctly empty collections on mongodb.
+- Command Generate - Catch correctly authentication error on mongodb.
+- Command Generate - Catch correctly empty collections on mongodb.
```

Comment:
Why do you have 4 lines of changes?
No auto-review?

Reply:
I did review my self But I did not had these 4 lines 🤔